libraries/habitat_shared.rb

```diff
@@ -16,26 +16,28 @@
 
 module Habitat
   module Shared
-    HAB_VERSION = '1.5.71'.freeze
-    LINUX_LAUNCHER_VERSION = '13458'.freeze
-    WINDOWS_LAUNCHER_VERSION = '13458'.freeze
-    WINDOWS_SERVICE_VERSION = '0.6.0'.freeze
+    # REMOVING THESE FOR TESTING
+    # No logical reason to force this versioning.
+    # HAB_VERSION = '1.5.86'.freeze
+    # LINUX_LAUNCHER_VERSION = '13553'.freeze
+    # WINDOWS_LAUNCHER_VERSION = '13553'.freeze
+    # WINDOWS_SERVICE_VERSION = '0.6.0'.freeze
 
-    def hab_version
-      HAB_VERSION
-    end
+    # def hab_version
```

Comment:
Why not delete if the code is dead?

Reply:
Whoa!!! I thought this was commented out already. good catch!

Reply:
fixed